tests/test65.c

```diff
@@ -16,19 +16,6 @@ struct CacheEntry {
 };
 struct CacheEntry *cache = NULL;
 
-static char * /*value*/ find_in_cache(const char *key)
```

Comment:
> The lines-of-code size of the checked-in code for test87.ans does matter to me

Seriously? We're not living in the age of 2M RAM and 14.4k dialup.
You're really concentrating on the wrong issue here.

You're right when you're saying there can be a more extensive test of the new macros. That unit test was from before your additions though, and was then sufficient.